src/Components/MainArea/Bookshelf/Bookcase/Bookcase.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import { IBooks } from '../../../../typings/IBooks';
+
+interface IProps {
```

Comment:
`IProps` sounds like a very generic name. Let's try to find a name that indicates the actual meaning of this interface. Maybe `IBookProps`?